gpflow/multikernels.py

```diff
@@ -1,25 +1,40 @@
 import tensorflow as tf
 
+from . import settings
 from .decors import params_as_tensors
 from .features import InducingPoints, InducingFeature
 from .kernels import Kernel, Combination
 from .params import Parameter
 
 
-class IndependentMultiKernel(Kernel):
+class MultiKernel(Kernel):
     pass
 
 
-class MultiKernel(Kernel):
+class IndependentMultiKernel(MultiKernel):
     pass
 
 
 class IndependentFeature(InducingFeature):
     pass
 
 
+class MultiInducingPoints(InducingPoints):
```

Comment:
Maybe I'm missing something but how is this different from the InducingPoints base class?